pymc3/distributions/posterior_predictive.py

```diff
@@ -224,7 +293,9 @@ def fast_sample_posterior_predictive(trace: Union[MultiTrace, Dataset, List[Dict
         if TYPE_CHECKING:
             _ETPParent = UserDict[str, np.ndarray]  # this is only processed by mypy
         else:
-            _ETPParent = UserDict  # this is not seen by mypy but will be executed at runtime.
+            _ETPParent = (
+                UserDict  # this is not seen by mypy but will be executed at runtime.
+            )
```

Comment:
if the comment was placed before the line it refers to, the formatting would be a lot nicer.

Reply:
done.